homeassistant/components/solarlog/sensor.py

```diff
@@ -36,10 +35,9 @@ class SolarLogSensorEntityDescription(SensorEntityDescription):
 
 SENSOR_TYPES: tuple[SolarLogSensorEntityDescription, ...] = (
     SolarLogSensorEntityDescription(
-        key="time",
+        key="last_updated",
```

Comment:
Why did you change the key?

As the key is used for the unique id, you can only change it if you provide also a unique_id migration for the old entities

Reply:
Is it ok to add the migration in `async_migrate_entry` (below L49)? 
When searching for examples, I found several integrations performing the migration in `async_setup_entry`, which I deem less appropriate in the case at hand.

Reply:
@edenhaus : Made a suggestion. Let me know your thoughts.